Monitoring in a Box/Ansible/roles/mib_setup/tasks/set_expire_password.yaml

```diff
@@ -0,0 +1,2 @@
+- name: set password to expired
+  shell: passwd -e "{{NewVmUsername}}"
```

Comment:
Use the `user` module with the `expires` parameter.

Reply:
Did try to use `user` module, but the `expires` parameter will target the user account, not the password.

Reply:
I see. Good find.